### PR TITLE
Add support for background color in convert

### PR DESCRIPTION
**Pull request by pentode**
Addresses #63

### CODE REVIEW

**Comment by enagorny** [comment]
@elad Could merge it please?

**Comment by dustingraves** [comment]
What is the status of this issue?

**Comment by elad** [comment]
@dustingraves if you can test it and verify that it works, I'll merge.

File: src/imagemagick.cc [hunk 1]
```diff
@@ -546,6 +562,10 @@ NAN_METHOD(Convert) {
     context->filter = !filterValue->IsUndefined() ?
         *NanAsciiString(filterValue) : "";
 
+    Local<Value> backgroundValue = obj->Get( NanNew<String>("background") );
```
Comment (dustingraves) [inline src/imagemagick.cc]: should this be `Nan::New` and not `NanNew`?

**Comment by dustingraves** [comment]
So in looking at the code it looks like it was written for `nan` 1.7 and now the new code is using 2.x. I am running OS X 10.12 and was not able to run the code because I got a bunch of errors with depricated V8 features in `nan` 1.7. I did a merge with the upstream master from elad and was able to get it running. However it did not appear change based on the background setting. I tried a couple variants for values i.e.: 'red', '#ff0000' etc. No changes reflected in the output images when running the tests with the added `background` value.

**Comment by dustingraves** [comment]
@pentode what is the expected result of this?

**Comment by rchipka** [comment]
+1

**Comment by rchipka** [comment]
@dustingraves @elad Please review #140 which is an actual working version with tests

**Comment by elad** [comment]
Fixed by #140

**Pull request closed without merging**